MediaWikiTelegramBot.js

```diff
@@ -172,18 +223,75 @@ tgBot.onText( /^([^\/].*)/, function ( msg, match ) {
             chatMessage,
             'Made with Telegram Bot',
             () => {
-                tgBot.sendMessage( userID, 'Translation published' );
-
-                userStatus[userID]['currentMwMessageIndex']++;
-                var nextMwMessage = getCurrentMwMessage( userID );
-
-                if ( nextMwMessage ) {
-                    tgBot.sendMessage(
-                        userID,
-                        nextMwMessage.definition
-                    );
-                }
+                // Remove that message from the list and set index to 0
+                // TODO - Update indexes in the array and the indexes shown to user after
+                delete ( userStatus[userID]['messages'][currentMwMessageIndex] );
+                currentMwMessageIndex = 0;
+
+                // Compile and display a message to the user
+                var message = 'Translation published\n';
+                message = message +
+                    "Here are the remaining messages:\n" +
+                    getMessages( userID ) +
+                    "\nType\n '/translate [message number] to choose a new message to translate." +
+                    "\nOr translate message 0:" +
+                    userStatus[userID]['messages'][0].definition;
+                tgBot.sendMessage( userID, message );
+
+                // userStatus[userID]['currentMwMessageIndex']++;
```

Comment:
@amire80 yeah, currently, I'm using messageIndex as a way to get which message the user wants to translate. Incorrect translations feature may not happen today... :p